This is now correctly failing on a warning:

```
Warning: /workflows/dependent-issues.yml:20:27: [warning] too few spaces before comment (comments)
Error: Process completed with exit code 2.
```

--- a/.github/workflows/dependent-issues.yml
+++ b/.github/workflows/dependent-issues.yml
@@ -17,7 +17,7 @@ on:
       - reopened
       - synchronize
   schedule:
-    - cron: '0 0/6 * * *' # every 6 hours
+    - cron: '0 0/6 * * *'  # every 6 hours
 
 jobs:
   check:

